app/controllers/projects_controller.rb

```diff
@@ -26,5 +26,8 @@ def show
       @issues_by_tag[:unassigned] << issue unless assigned
     end
 
+    @count_by_tag = @issues_by_tag.map do |tag, issues|
```

Comment:
Can't you initialize this like we do in L13 and then reuse the previous loop to update the count (at the same time you update the `@issues_by_tag` in L22 and L26?